lib/matplotlib/pyplot.py

```diff
@@ -429,6 +429,7 @@ def figure(num=None,  # autoincrement if None, else integer from 1-N
            facecolor=None,  # defaults to rc figure.facecolor
            edgecolor=None,  # defaults to rc figure.edgecolor
            frameon=True,
+           clf=False,
```

Comment:
I would call it `'clear'`, because `clf` has a slightly different meaning in the pyplot context.  'clear' is clearer.